src/modules/app/containers/NavigatorContainer.js

```diff
@@ -3,15 +3,24 @@
 import { type Dispatch } from 'redux'
 import { connect } from 'react-redux'
 import Navigator from '../components/Navigator'
+import type { StoreActionType } from '../StoreActionType'
 
 type DispatchPropsType = {|
+  setContentLanguage: (language: string) => void,
   fetchCategory: (cityCode: string, language: string, key: string) => void,
-  clearCategory: (key: string) => void
+  clearCategory: (key: string) => void,
+  fetchCities: () => void
 |}
 
 type PropsType = DispatchPropsType
 
-const mapDispatchToProps = (dispatch: Dispatch<*>): DispatchPropsType => ({
+const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>): DispatchPropsType => ({
+  setContentLanguage: (language: string) => {
+    dispatch({
```

Comment:
This is not necessary, since I already typed the dispatch function with `Dispatch<StoreActionType>` which in turn only allows an action of this type to be dispatched.